modules/post/windows/gather/credentials/filezilla_server.rb

```diff
@@ -55,26 +55,28 @@ def check_filezilla
     end
 
     session.fs.dir.foreach(path) do |fdir|
-      if fdir =~ /FileZilla\sServer.*\.xml/i
-        paths << path + fdir
+      ['FileZilla Server.xml','FileZilla Server Interface.xml'].each do|xmlfile|
+        if fdir.eql? xmlfile
+          pathtmp = File.join(path + xmlfile)
+          vprint_status("Configuration File Found: %s" % pathtmp)
```

Comment:
Yeah. This convention is used in a few places outside this PR as well. I believe the original author was a python programmer as well!.